test/integration/features/basic.feature

```diff
@@ -8,6 +8,14 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
```

Comment:
@gbraad I agree, I added some comment so that we are clear what we want to achieve here.